src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -1128,9 +1128,31 @@ namespace OpenLoco::ObjectManager
     }
 
     // 0x0047AC05
+    // Initialise lastTrackTypeOption in game state
     void sub_47AC05()
     {
-        call(0x0047AC05);
+        static_assert(ObjectManager::getMaxObjects(ObjectType::road) <= 128); // protect against possible int8_t overflow in the future
```

Comment:
128 is actually an issue as that sets the high bit it should be `<= 127` or `< 128`